src/components/Typography/index.stories.tsx

```diff
@@ -20,6 +20,13 @@ export const Canvas = () => (
     label={text("label", "Typography example")}
     truncated={boolean("truncated", false)}
     variant={select("variant", TypographyVariants, TypographyVariants.body)}
+    display={select(
+      "display",
+      TypographyDisplay,
+      TypographyDisplay.default,
```

Comment:
Remove lines below `TypographyDisplay.default` they won't be interpreted by storybook select addon.